### ISSUE TITLE
More fully document the React dependency

### ISSUE THREAD

**Issue by curtcox** [dxos/dxos#5521]
As someone just starting, I would like to understand the React dependency more.
- What does it provide?
- Would a vanilla HTML and Javascript example be possible?
- Would a htmx example be possible?

**Comment by wittjosiah** [comment]
@curtcox can I ask what information on the React client you're looking for specifically? Are you looking for api docs on `@dxos/react-client`? If so, I suggest following this issue https://github.com/dxos/dxos/issues/5126 as we're tracking those being missing there.

As far as a more vanilla web dev experience, we've gotten that feedback before and have an issue for it https://github.com/dxos/dxos/issues/4617.

As far as htmx is concerned, that's not something that I've used myself but I can't see any reason it wouldn't be possible. We only provide an official react client, but the typescript client can be integrated with any frontend framework.

**Comment by curtcox** [comment]
@wittjosiah I'm trying to understand the relationship between DXOS and React. I see React Tutorial and React Guide as two of the main headings in the [guide](https://docs.dxos.org/guide/getting-started.html). 
This gives me the impression that React is somehow required. If it is required, then I would like to understand why. I'm hoping that there is no requirement and is so prominent in the docs because of its popularity. Your statement that "the typescript client can be integrated with any frontend framework" sounds like that is the case. If the independence is already in the guide, you might want to make it more explicit. It could be that all of the information I want is already there if I had completely read the docs.

**Comment by wittjosiah** [comment]
@curtcox ah I see, thank you for reaching out to provide us with that feedback.

I'll point out that the quick start guide you link to states:

> DXOS works in any Node.js or Browser environment. There is a [TypeScript API](https://docs.dxos.org/guide/typescript) and a [react API](https://docs.dxos.org/guide/react).

We've distinguished usage with React from plain Typescript usage throughout the docs along these lines. That being said, the docs still need _a lot_ of work, so we'll try to incorporate this feedback into future improvements to the docs.

I'm going to close this issue in an attempt to keep the issues list from becoming unmanageable, but I've made note of this concern in https://github.com/dxos/dxos/issues/3707. Feel free to note any additional confusions you have with the docs there and the team will use that feedback to inform our docs work.
